### DISCUSSION TITLE
AM550 config for Austria Wiener Netze

### DISCUSSION THREAD

**Discussion by zeha** [UtilitechAS/amsreader-firmware#825] [General]
The grid operator for Vienna ("Wiener Netze") has rolled out various smartmeters, including the Iskraemeco AM550. However, it is accessible only via Infrared, and the datagram is relatively short.

Grid operator doc: https://www.wienernetze.at/kundenschnittstelle2#14649034:~:text=Kundenschnittstelle,-Der

From what I get from my meter, it appears its really just this:

```
02 09
09 0C 07 E8 08 03 06 10 04 1B 00 FF 88 80  // 0-0:1.0.0 date time
06 00 0F 52 0D  // 1-0:1.8.0 Active energy A+ (consumed)
06 00 00 00 01  // 1-0:2.8.0 Active energy A- (return)
06 00 00 00 0C  // 1-0:3.8.0 Reactive energy +R
06 00 1F D3 90  // 1-0:4.8.0 Reactive energy -R
06 00 00 03 27  // 1-0:1.7.0 instantaneous power P+
06 00 00 00 00  // 1-0:2.7.0 instantaneous power P-
06 00 00 00 00  // 1-0:3.7.0 instantaneous power Q+
06 00 00 01 69  // 1-0:4.7.0 instantaneous power Q-
```

Is there a way of supporting this setup, or is it hopeless because there is no way to ID this automatically?

**Comment by zeha** [comment]
https://github.com/UtilitechAS/amsreader-firmware/pull/826 works for me

**Comment by ArnieO** [comment]
It is a bit concerning that there are Iskra AM550 meters that can only be read through the optical port. 

Other version of the AM550 has a "real" (electrical) user port according to [DSMR P1 Companion Standard](https://www.netbeheernederland.nl/_upload/Files/Slimme_meter_15_a727fce1f1.pdf), and can be read using a Pow-P1.

**Reply by zeha** [reply]
The exact availability of ports is probably customer-specific (where customer = grid operator).

**Reply by ArnieO** [reply]
I probably is, but could cause problems if someone uses for instance our [Selection guide](https://www.amsleser.no/index.php?fc=module&module=ams_selector&controller=display) to find the right product. 

IMO they should have had separate model numbers.
